xcodeproj/internal/automatic_target_info.bzl

```diff
@@ -200,6 +206,13 @@ def calculate_automatic_target_info(ctx, build_mode, target):
         is_top_level = True
         provisioning_profile = "provisioning_profile"
         xcode_targets = _TEST_BUNDLE_XCODE_TARGETS
+    elif _is_static_framework(target):
+        # TODO: https://github.com/MobileNativeFoundation/rules_xcodeproj/issues/105
+        # Static frameworks are not supported yet as Xcode targets.
+        is_supported = False
+        is_top_level = True
+        collect_uncategorized_files = rule_kind != "apple_bundle_import"
+        xcode_targets = _DEFAULT_XCODE_TARGETS[this_target_type]
```

Comment:
Are these the appropriate fields to forward?

Reply:
Good Q and also not sure, maybe @brentleyjones could chime in?

Reply:
It's a good start. We will know if we need more fields.